### DISCUSSION TITLE
Is there a way to prefer Trakt ratings over Plex ratings?

### DISCUSSION THREAD

**Discussion by stathisstef** [Taxel/PlexTraktSync#1761] [Q&A] [answered]
Currently, if I change the rating of an already rated movie in my Plex library, this new rating will sync over to Trakt without an issue. But if I change the rating in Trakt, my new Trakt rating is replaced by the old Plex rating on the next sync.

I imagine this is by design, but is there any way to make it so that it prefers Trakt ratings instead of Plex ratings in the case of a discrepancy?

**Comment by simonc56** [comment, marked as answer]
Check `rating_priority` in your *config.yml* file.

**Reply by stathisstef** [reply]
Thank you! I was searching in the wrong place – just found the feature request thread

**Reply by glensc** [reply]
And as for all:

1. link that thread of others to see what you found if they come up to this discussion in the future (don't be selfish)
2. you can contribute a mr to add changes to readme (help with the project poor documentation).
